api/requirements.txt

```diff
@@ -80,3 +81,10 @@ lxml==5.1.0
 xlrd~=2.0.1
 pydantic~=1.10.0
 pgvecto-rs==0.1.4
+xlrd~=2.0.1
+pdfminer==20191125
+pdfminer.six==20201018
+pypdf==4.2.0
+PyMuPDF==1.24.0
```

Comment:
Seems like some dependencies requiring commercial licenses have been introduced, please check it out. 🧐